cc/mcts_node.cc

```diff
@@ -186,9 +186,24 @@ void MctsNode::IncorporateResults(absl::Span<const float> move_probabilities,
     return;
   }
 
+  float policy_scalar = 0;
+  for (int i = 0; i < kNumMoves; ++i) {
+    if (!illegal_moves[i]) {
+        policy_scalar += move_probabilities[i];
+    }
+  }
+  if (policy_scalar > std::numeric_limits<float>::min()) {
+    policy_scalar = 1 / policy_scalar;
+  }
+
+  std::cout << "What what " << policy_scalar << std::endl;
```

Comment:
Assuming this was for debugging? :D

Reply:
Yes, I need to re-push the new version with tests.